Sources/Sentry/SentrySession.m

```diff
@@ -39,7 +42,8 @@ - (instancetype)initDefault
         _status = kSentrySessionStatusOk;
         _sequence = 1;
         _errors = 0;
-        _distinctId = [SentryInstallation id];
+        SentryOptions *options = [[[SentrySDK currentHub] getClient] options];
+        _distinctId = [SentryInstallation idWithOptions:options];
```

Comment:
`m`: `initWithReleaseName` is called from the hub where we can access the options. So instead of using `[[SentrySDK currentHub] getClient] options];` we should ask for the cacheDirectoryPath in `initWithReleaseName`, so we can remove reference to the `SentrySDK` here.